google/ggrc-core
# GGRC-8257 Filter by state in Bulk Verify pop up should be disabled

# Issue description

Filter by state in Bulk Verify pop up should be disabled. 

# Steps to test the changes

1. Open any Audit.
2. Create Assessment.
3. Add verifier and move Assessment to In Review state.
4. Open My Assessments.
5. Click Bulk Verify button.
6. Verify that  State field is be disabled with 'In Review' state. Operator field is not shown. 

# Solution description

Set `false` for `showOperator`  and `disabled` to hide Operator field  and to disable Filter by State. 

# Sanity checklist




# PR Review checklist
